src/App.css

```diff
@@ -1,38 +1,108 @@
-.App {
+@font-face {
+  font-family: "Open Sans";
+  font-style: normal;
+  font-weight: 400;
+  src: local("Open Sans"), local("OpenSans"),
+    url(https://fonts.gstatic.com/s/opensans/v13/cJZKeOuBrn4kERxqtaUH3T8E0i7KZn-EPnyo3HZu7kw.woff)
+      format("woff");
+}
+
+.bg {
```

Comment:
Have you decided not to use Tailwind? Why was that?

Reply:
No, hybrid is not a good idea at all, either use one or the other. Main reason I asked, is because there are issues with the CSS that would likely be avoided by using Tailwind, but if you prefer to use CSS, we can review that.